src/pjpersist/serialize.py

```diff
@@ -66,6 +66,9 @@
     '__builtin__.set': set,
 }
 
+DICT_NON_STRING_KEY_MARKER = 'dict_data'
```

Comment:
Let's call it `'non_string_key_marker'` instead. `dict_data` doesn't make any sense, if it is not dictionary data.

Reply:
This edge case is dict only. JSON(B) accepts only strings as keys.

Reply:
What does this have to do with `dict_data`?